src/styles/colors.js

```diff
@@ -42,8 +45,10 @@ const darkModeColors = {
   whiteLabel: '#FFFFFF',
 };
 
-const isHex = (color = '') => color.length >= 3 && color.charAt(0) === '#';
```

Comment:
Are you 100% sure that checking for length and comparing character is actually slower than building a key and looking in a dictionary? Do you have some performance stats?